ethereum/p2p/src/main/java/tech/pegasys/pantheon/ethereum/p2p/rlpx/connections/netty/NettyConnectionInitializer.java

```diff
@@ -165,7 +169,7 @@ public void subscribeIncomingConnect(final ConnectCallback callback) {
     final CompletableFuture<PeerConnection> connectionFuture = new CompletableFuture<>();
 
     if (peer instanceof DiscoveryPeer) {
-      ((DiscoveryPeer) peer).setLastAttemptedConnection(System.currentTimeMillis());
+      ((DiscoveryPeer) peer).setLastAttemptedConnection(clock.millis());
```

Comment:
This line duplicates logic in `RlpxAgent`, so we can cut it here and cut the `Clock` dependency as well.